Glob expressions in scripted do not support absolute paths

## steps

As seen in https://github.com/jtjeferreira/sbt-native-packager/actions/runs/12422203739/job/34684163992

```
Caused by: java.lang.IllegalArgumentException: Couldn't create relativeGlob glob from absolute glob: /usr/lib/systemd/system/rpm-test.service
	at sbt.nio.file.RelativeGlob$.parse(Glob.scala:710)
	at sbt.nio.file.Glob$.apply(Glob.scala:107)
	at sbt.internal.scripted.FileCommands.filterFromStrings$$anonfun$1(FileCommands.scala:75)
	at scala.collection.immutable.List.map(List.scala:247)
	at sbt.internal.scripted.FileCommands.filterFromStrings(FileCommands.scala:75)
	at sbt.internal.scripted.FileCommands.exists(FileCommands.scala:114)
	at sbt.internal.scripted.FileCommands.commandMap$$anonfun$3(FileCommands.scala:28)
	at scala.runtime.function.JProcedure1.apply(JProcedure1.java:15)
	at scala.runtime.function.JProcedure1.apply(JProcedure1.java:10)
	at sbt.internal.scripted.FileCommands.nonEmpty$$anonfun$1(FileCommands.scala:142)
	at scala.runtime.function.JProcedure1.apply(JProcedure1.java:15)
	at scala.runtime.function.JProcedure1.apply(JProcedure1.java:10)
	at sbt.internal.scripted.FileCommands.apply$$anonfun$1(FileCommands.scala:49)
	at scala.runtime.function.JProcedure1.apply(JProcedure1.java:15)
	at scala.runtime.function.JProcedure1.apply(JProcedure1.java:10)
	at scala.Option.map(Option.scala:242)
	at sbt.internal.scripted.FileCommands.apply(FileCommands.scala:49)
	at sbt.internal.scripted.BasicStatementHandler.apply(StatementHandler.scala:25)
	at sbt.internal.scripted.BasicStatementHandler.apply$(StatementHandler.scala:20)
	at sbt.internal.scripted.FileCommands.apply(FileCommands.scala:19)
	at sbt.internal.scripted.FileCommands.apply(FileCommands.scala:19)
	at sbt.scriptedtest.BatchScriptRunner.processStatement(BatchScriptRunner.scala:48)
```

The problematic line being 

```
$ exists /usr/lib/systemd/system/rpm-test.service
```

Relates to https://github.com/sbt/sbt/pull/7932 and https://github.com/sbt/sbt/pull/7933

--- a/internal/util-scripted/src/main/scala/sbt/internal/scripted/FileCommands.scala
+++ b/internal/util-scripted/src/main/scala/sbt/internal/scripted/FileCommands.scala
@@ -56,6 +56,11 @@ class FileCommands(baseDirectory: File) extends BasicStatementHandler {
   def fromStrings(paths: List[String]) = paths.map(fromString)
   def fromString(path: String) = new File(baseDirectory, path)
   def filterFromStrings(exprs: List[String]): List[PathFilter] = {
+    def globs(exprs: List[String]): List[PathFilter] =
+      exprs.map { g =>
+        if (g.startsWith("/")) (Glob(g): PathFilter)
+        else (Glob(baseDirectory, g): PathFilter)
+      }
     def orGlobs = {
       val exprs1 = exprs
         .mkString("")
@@ -63,19 +68,18 @@ class FileCommands(baseDirectory: File) extends BasicStatementHandler {
         .filter(_ != OR)
         .toList
         .map(_.trim)
-      val combined = exprs1.map(Glob(baseDirectory, _)) match {
+      val combined = globs(exprs1) match {
         case Nil      => sys.error("unexpected Nil")
-        case g :: Nil => (g: PathFilter)
+        case g :: Nil => g
         case g :: gs =>
-          gs.foldLeft(g: PathFilter) {
-            case (acc, g) =>
-              acc || (g: PathFilter)
+          gs.foldLeft(g) {
+            case (acc, g) => acc || g
           }
       }
       List(combined)
     }
     if (exprs.contains("||")) orGlobs
-    else exprs.map(Glob(baseDirectory, _): PathFilter)
+    else globs(exprs)
   }
 
   def touch(paths: List[String]): Unit = IO.touch(fromStrings(paths))

--- a/sbt-app/src/sbt-test/actions/add-alias/test
+++ b/sbt-app/src/sbt-test/actions/add-alias/test
@@ -2,3 +2,5 @@
 -> demo-failure
 > +z
 > z
+
+$ absent /tmp/non-existent

